src/signalrclient/json_helpers.cpp

```diff
@@ -57,7 +58,42 @@ namespace signalr
             // because the server expects certain values to be 1 instead of 1.0 (like protocol version)
             if (std::modf(value, &intPart) == 0)
             {
-                return Json::Value((int)intPart);
+				if (value < 0)
```

Comment:
Nit: Use spaces instead of tabs.

Reply:
Converted all tabs to spaces in subsequent submissions.